use-cases.md

```diff
@@ -69,11 +65,7 @@ The `operation` above refers to the name the server wants to give to the Span. F
 The `span` object above can be `None` if the Tracer did not find relevant headers in the incoming request: presumably because the client did not send them. In this case the server needs to start a brand new trace.
 
 ```python
-    span = tracer.join_trace_from_text(
-        operation_name=operation,
-        context_snapshot=request.headers,
-        trace_attributes=request.headers
-    )
+    span = tracer.get_span_extractor(request.class).extract(request)
```

Comment:
> I think what you're saying is that the custom end-user written codec will not be an f(span, target) but an f(span, target, g)  where g = tracer.injector(map or binary)

No. There is no need to add a parameter, especially since this is not a per call argument.
My presumption is that the injector already holds reference to the tracer, and therefore can obtain the injector to delegate to, by itself.